userguide/storage.rst

```diff
@@ -2488,7 +2476,8 @@ Replication depends on SSH, disks, network, compression, and
 encryption to work. A failure or misconfiguration of any of these can
 prevent successful replication.
 
-Replication logs are located in :file:`var/log/zettarepl.log`.
+All replication logs are located in :file:`/var/log/zettarepl.log`. You can
```

Comment:
Avoid "you" or "your".
```suggestion
Replication logs are saved in :file:`/var/log/zettarepl.log` and can be
```

Reply:
Is it too late to change the name of the log file to a generic `replication.log`?  Using the name of a particular implementation in the log filename will often become a hassle down the road, like if the replication system changes.  replication.log would work for anything.

Reply:
User won't need to interact with this file, we have logs in UI that are split per-task. I've named it after zettarepl because that's how other files in /var/log are named: nginx/access.log, not webserver/access.log; proftpd.log, not ftp.log, etc. If the replication system changes, replication.log will change it's format too and being on the same place won't give nobody any help.